### PR TITLE
Implement the UI for the Create Competition page.

### PR DESCRIPTION
**Pull request by Phoebe-Khokgawe**
Implemented the UI for the Create Competition Page.
For now adding the participants user has to type out an email in the text box and click the add button.
This is how the page looks 
image

The plan is to displayed the entered email when the add button is clicked.

### CODE REVIEW

**Comment by lin8231** [comment]
Could you make the bottom buttons the same size?

**Comment by lin8231** [comment]
Can you also either left align or right align the field titles?
Here:
Screen Shot 2021-02-02 at 12 36 49 pm

**Comment by Phoebe-Khokgawe** [comment]
> Can you also either left align or right align the field titles?
> Here:
> Screen Shot 2021-02-02 at 12 36 49 pm



> Could you make the bottom buttons the same size?

fixed here 2fbc285394c8a3f5e5c5a402931dff000c10aaaf
image

File: project/frontend/src/containers/CreateCompetition/CreateCompetition.js [hunk 1]
```diff
@@ -1,68 +1,45 @@
...
           </div>
         </div>
-        <div className={classes.CompetitiomInfoRight}>
+        <div className={classes.CompetitionInfoRight}>
           <p>Participants</p>
+          <div className={classes.AddParticipants}>
+            <p>
+              {" "}
+              Add Participants: <input type="text" />{" "}
+              <LinkButton inverted="true">Add</LinkButton>
```
Comment (texm) [inline project/frontend/src/containers/CreateCompetition/CreateCompetition.js:30]: Why is this a `LinkButton`? Adding a participant shouldn't change page history, right?

Comment (Phoebe-Khokgawe) [reply project/frontend/src/containers/CreateCompetition/CreateCompetition.js:30]: No it doesn't change the page it just display the email on the space below. I'm only using it cause the style match the others but I can definitely create my own button if that will make it more efficient.

Comment (texm) [reply project/frontend/src/containers/CreateCompetition/CreateCompetition.js:30]: Should be fine, just a naming thing.

**Review by texm — APPROVED** [review]

**Review by lin8231 — APPROVED** [review]

**Pull request merged**